rust/relay/src/allocation.rs

```diff
@@ -100,7 +100,7 @@ async fn forward_incoming_relay_data(
         tokio::select! {
             result = socket.recv() => {
                 let (data, sender) = result?;
-                relayed_data_sender.send((data.to_vec(), PeerSocket::new(sender), id)).await?;
```

Comment:
Here used to be the 1. allocation. Note that `PeerToClient::new` still allocates.